lib/profile/view_profile/sections/contact.dart

```diff
@@ -137,6 +143,8 @@ class ProfileContact extends StatelessWidget {
         builder: (context) => ChatMessenger(
           recipient: user.fullName,
           peerId: user.id,
+          groupChatId: matchId,
+          id: id
         )
```

Comment:
I'm kind of finding a hard time trying to differentiate whose id is whose here... is the `id` prop the sender's id? If so, can you rename it so that it is `senderId: id`? Also maybe pass the whole user object as the recipient since you're gonna need both the fullname and id anyways? So something like:
```
    ChatMessenger(
          recipient: user,
          groupChatId: matchId,
          senderId: id
     )
```